source/administration/command-line-tools.rst

```diff
@@ -420,35 +420,26 @@ platform license upload
 
       sudo ./platform license upload /path/to/license/mylicensefile.mattermost-license
 
-platform message_export
+platform export
 ------------------------
 
-  Description
-    Commands to run compliance exports.
-
-  Child Command
-    -  `platform message_export export`_ - Export data from Mattermost.
-
-platform message_export export
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
   Description
     Export data from Mattermost in a format suitable for importing into a third-party archive system.
 
   Format
     .. code-block:: none
 
-      platform message_export export
+      platform export
 
   Example
     .. code-block:: none
 
-      sudo ./platform message_export export --format=actiance --exportFrom=1513102632
+      sudo ./platform export --format=actiance --exportFrom=1513102632
```

Comment:
We also support an `--timeoutSeconds` option that lets the user dictate how long the export should run for before timing out. It's in the `--help` for the CLI command. Not sure if it's worth explicitly mentioning here or not.

Reply:
Added, it's worth including it in the doc